### DISCUSSION TITLE
Improve OpenGraph metadata for linking to discussions

### DISCUSSION THREAD

**Discussion by aschrijver** [community/community#3232] [Discussions]
[This discussion](https://github.com/pubpub/pubpub/discussions/1249) when linked in [this forum post](https://meta.discourse.org/t/community-has-no-boundary-discourse-as-a-fabric-ideation-brainstorm/174578/18), generates the following OpenGraph link preview on Discourse forums:

![github-discussions-improve-opengraph-metadata](https://user-images.githubusercontent.com/5111931/105821925-b11e2d00-5fbb-11eb-8099-c8b650ff4282.png)

Which can be improved by at least generating a good `og:description` meta tag.

**Comment by michaelsena** [comment]
+1 for this
